### PR TITLE
Share files with participants in call + URL speed dial + CSS fixes

### LINKED ISSUE #23: Share files with participants in call
**Describe the feature you'd like**
Easy way to share files with people I'm in a call with.

**Additional context**
WebRTC supports sending arbitrary data over the wire. Should be possible to chunk up a file and send it over the data channels. We want functionality similar to https://www.sharedrop.io/

### PR DESCRIPTION
**Pull request by MarcusMiguel**
Closes #23 and #8 

**Share files with participants in call:**
Screenshot_1
Screenshot_2

**Permanent paths for calling ships:**
Going to **/apps/campfire/call/zod** redirects to a call with ~zod

Also includes the changes made at #31

### CODE REVIEW

**Comment by datryn-ribdun** [comment]
Hey @MarcusMiguel,
Thanks for making this PR! Looks like you addressed some issues that were high on my list of wants, so thanks for that.
Give me a few days to review.

**Comment by datryn-ribdun** [comment]
I think we should take the File Transfer UI out of Controls.tsx, and add it as part of the chat box:
1. Easier to tell that someone is trying to send you a file (otherwise how would someone know to click on the Files button)
2. Cleaner interface leaving the video controls to just video stuff + hangup (this is subjective)
3. Adds context to file transfers. Ex: ~zod asks for the latest podcast episode, ~bus responds with a request to share "urbit-podcast.mp3".

Do you think this is reasonable/better design?

**Comment by datryn-ribdun** [comment]
Note for me: Need to set up pre-commit hooks so that `npm run lint` runs and passes before commits go through. linting would catch some of the easy suggestions I'm making.

**Comment by MarcusMiguel** [comment]
Adding it to the chat box was my original idea, i kinda gave up because i was struggling to think in a good way to add the button there (after seeing the Realm chat i think that it could be done like that).

The file transfer in the middle of the screen has the advantage that its easier to browse/read things there.

Note: the file transfer menu will open automaticaly when someone sends a file.

Another thing that i forgot to mention is that i started working on #2 (but its responsive only in the home screen, i dont even remember how it looks like, lol).

**Comment by MarcusMiguel** [comment]
Not sure if I included the responsiveness changes in this PR.

File: campfire/ui/src/components/ShareFileDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+                                            <Text >Drop files here</Text>
+                                            <Text >or</Text>
+                                            <Text fontWeight='medium'>Click to browse</Text>
+                                        </>
+
+                                    }
+                                    {
+                                        files && files.length === 1 &&
+                                        <>
+                                            <MdUploadFile className="w-8 h-8 mb-2" style={{ color: 'var(--rlm-icon-color, #000000)', opacity: '0.5', fontSize: '20px' }} aria-hidden />
```
Comment (datryn-ribdun) [inline campfire/ui/src/components/ShareFileDialog.tsx:315]: nice job using the `var(--rlm-icon-color, #000000)`. Thats the code that makes realm theming work

**Comment by datryn-ribdun** [comment]
> Adding it to the chat box was my original idea, i kinda gave up because i was struggling to think in a good way to add the button there (after seeing the Realm chat i think that it could be done like that).
> 
> The file transfer in the middle of the screen has the advantage that its easier to browse/read things there.
> 
> Note: the file transfer menu will open automaticaly when someone sends a file.
> 

I think we can keep it as is to get this merged, but I'll file an issue that the ideal design would be having it in the chat box.

**Comment by MarcusMiguel** [comment]
Maybe another issue is that we are using two file icons from **react-icons**. I think that there is no file icons in the public design system.

**Comment by datryn-ribdun** [comment]
@MarcusMiguel please ask for another review once you've commited the changes requested.

**Comment by MarcusMiguel** [comment]
I think that this is ready for review.
I improved the mobile formatting on the Meeting Page too, but the video still small. 
Also, i dont have Realm so idk how this app looks with themes.

**Comment by datryn-ribdun** [comment]
Things look good and ready to merge.
Only weird bug, that might just be my computer (Fedora linux), is the file selector defaults to not letting my select most file extension types. I don't want to block merging on this, so I'll revisit it later.
I'll need to update the changelog and cut a release before this actually gets distributed on network, but that should only take 1-2 days.

**Comment by MarcusMiguel** [comment]
Weird, the file upload was accepting all files here (Windows). Even with the **accept** atribute.

**Pull request merged**